src/tests/ftest/launch.py

```diff
@@ -2184,6 +2223,18 @@ def main():
         action="store_true",
         help="when replacing server/client yaml file placeholders, discard "
              "any placeholders that do not end up with a replacement value")
+    parser.add_argument(
+        "-dsd", "--disable_stop_daos",
+        action="store_true",
+        help="disable stopping DAOS servers and clients between running tests")
```

Comment:
Just a design thought...
```suggestion
    parser.add_argument(
        "-sd", "--stop_daos",
        type=bool,
        default=True,
        help="Stop DAOS servers and clients between running tests")
```

Reply:
I was thinking something like `--stop_daos true` vs `--stop_daos false`, but I see your point. No problem by me